### PR TITLE
docs(framework): combine framework overview and bindings

### PR DESCRIPTION
**Pull request by rwaskiewicz**
this commit combines the framework overview and bindings pages. the intent of this change it to remove redundancy between these two pages.

I still need to propagate this to other versions of the docs, which I'll do once we have an initial round of approvals

### CODE REVIEW

File: vercel.json [hunk 1]
```diff
@@ -119,6 +119,11 @@
       "source": "/state-tunnel(/)?",
       "destination": "https://github.com/ionic-team/stencil-state-tunnel",
       "statusCode": 301
+    },
+    {
+      "source": "/framework-bindings",
+      "destination": "/docs/overview",
```
Comment (rwaskiewicz) [inline vercel.json:125]: This is the Frameowrk Overview page - which is...odd. I'll deal with this in a separate PR

**Review by tanner-reits — APPROVED** [review]
I like the change 👍

**Review by alicewriteswrongs — APPROVED** [review]
the `bindings` page struck me as a little redundant, good to eliminate it!

**Comment by rwaskiewicz** [comment]
@tanner-reits @alicewriteswrongs can you PTAL at https://github.com/ionic-team/stencil-site/pull/1050/commits/f5fb9970981793859ae1771e18f621ca473fd4f5? I've propagated those changes to the other versions

**Review by alicewriteswrongs — APPROVED** [review]

**Pull request merged**